zerver/views/user_settings.py

```diff
@@ -126,13 +126,13 @@ def json_change_settings(request: HttpRequest, user_profile: UserProfile,
 @has_request_variables
 def update_display_settings_backend(
         request: HttpRequest, user_profile: UserProfile,
-        twenty_four_hour_time: Optional[bool]=REQ(validator=check_bool, default=None),
-        high_contrast_mode: Optional[bool]=REQ(validator=check_bool, default=None),
-        night_mode: Optional[bool]=REQ(validator=check_bool, default=None),
-        default_language: Optional[bool]=REQ(validator=check_string, default=None),
-        left_side_userlist: Optional[bool]=REQ(validator=check_bool, default=None),
-        emojiset: Optional[str]=REQ(validator=check_string, default=None),
-        timezone: Optional[str]=REQ(validator=check_string, default=None)) -> HttpResponse:
+        twenty_four_hour_time: Optional[bool]=REQ(validator=check_bool, default=None, type=bool),
+        high_contrast_mode: Optional[bool]=REQ(validator=check_bool, default=None, type=bool),
+        night_mode: Optional[bool]=REQ(validator=check_bool, default=None, type=bool),
+        default_language: Optional[bool]=REQ(validator=check_string, default=None, type=bool),
```

Comment:
This one seems wrong; should be `type=str`.

Reply:
The `default_language` change was to match the parameter annotation, which was a previous edit by yourself (03b4a024) where the number of annotations in python2 style didn't match the number of parameters. So, if the parameter should be `Optional[str]` then `type=str` makes sense too.